generator/graphql-kotlin-federation/src/main/kotlin/com/expediagroup/graphql/generator/federation/types/_Service.kt

```diff
@@ -36,7 +36,7 @@ private val SERVICE_OBJECT_TYPE = GraphQLObjectType.newObject()
 
 internal val SERVICE_FIELD_DEFINITION: GraphQLFieldDefinition = GraphQLFieldDefinition.newFieldDefinition()
     .name("_service")
-    .type(GraphQLNonNull.nonNull(SERVICE_OBJECT_TYPE))
+    .type(GraphQLNonNull(SERVICE_OBJECT_TYPE))
```

Comment:
question: shouldn't `_service` be a `nonNull` field ? as fixed in this PR  https://github.com/ExpediaGroup/graphql-kotlin/pull/1460

Reply:
it is -> just changed to constructor invocation vs static factory to make it consistent with [service object type definition](https://github.com/ExpediaGroup/graphql-kotlin/blob/f75994d4865146866c750a3b33b477c23d34d5e1/generator/graphql-kotlin-federation/src/main/kotlin/com/expediagroup/graphql/generator/federation/types/_Service.kt#L32)